connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -68,6 +69,37 @@ message Relation {
     StatCrosstab crosstab = 101;
     StatDescribe describe = 102;
 
+    // Catalog API (internal-only)
```

Comment:
The better way of dealing with this is to wrap all of the catalog messages into a parent message. This makes it easier to reason about this and eventually remove if we want.

Reply:
Thanks Martin. Let me try to take a stab